### PR TITLE
Replaced old BLE scan library with BluePy scan code 

### PR DESCRIPTION
**Pull request by OrenLederman**

### CODE REVIEW

File: src/badge_discoverer.py [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
+					devices[mac] = {'scan_date':scan_date,'is_sync':is_sync,'rssi':rssi}
+				else:
+					devices[mac]['rssi']=rssi
+					devices[mac]['scan_date'] = scan_date
+
+		return devices
+
+class ScanDummy(btle.DefaultDelegate):
+    def handleDiscovery(self, dev, isNewDev, isNewData):
+    	None
```
Comment (adminq80) [inline src/badge_discoverer.py]: This line should be pass instead of None

File: src/badge_discoverer.py [hunk 2]
```diff
@@ -1,50 +1,50 @@
...
-import time
-
-# Looks for badges and return their state (synced or not), the 
-# date of the latest scan, and a list of RSSIs
-class BadgeDiscoverer:
-	def __init__(self):
-		self.DEVICE_NAME = "MiniBadge"
-		self.CLOCK_STATE_SYNC = "CLKSYN"
-		self.CLOCK_STATE_UNSYNC = "CLKUN"
-		None
```
Comment (adminq80) [inline src/badge_discoverer.py]: None is not needed

Comment (OrenLederman) [inline src/badge_discoverer.py]: Fixed

**Comment by adminq80** [comment]
Looks good

**Pull request merged**